resources/compass/templates/migrator-job.yaml

```diff
@@ -60,3 +73,9 @@ spec:
                   args:
                     - "-c"
                     - "./run.sh; exit_code=$?; echo '# KILLING PILOT-AGENT #'; curl -XPOST http://127.0.0.1:15020/quitquitquit; sleep 5; exit $exit_code;"
```

Comment:
If `cloud_sql_proxy` doesn't have a dedicated endpoint, please use the `pkill -INT` command like we did before `quitquitquit` endpoint for Istio sidecar: https://github.com/kyma-project/kyma/pull/6343/files#diff-44b7cd4a24bf84a8d012823ad10723a6L19

Reply:
Has updated the solution again using pkill, gcr.io/cloudsql-docker/gce-proxy:1.11 is not working with pkill -INT well, using gcr.io/cloudsql-docker/gce-proxy:1.13